client/__tests__/test.js

```diff
@@ -0,0 +1,26 @@
+const sum = require('../sum.js');
```

Comment:
You've just been working on testing so I wont' say much here other than...why is your postgreSQL test in your client?

Reply:
fixed .. thanks!